### DISCUSSION TITLE
How should I add a missing config for Everspring TST01-1 Flood Detector?

### DISCUSSION THREAD

**Discussion by mnaz** [zwave-js/zwave-js#1858] [Q&A] [answered]
This sensor is not recognized with node-zwave-js

Everspring TST01-1 Flood Detector
https://products.z-wavealliance.org/products/736

Closest Device Configuration I could find
https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0060/st812.json

This doc seems to show the format
https://zwave-js.github.io/node-zwave-js/#/development/config-files

Should I just try and add my sensor product type/id values to the existing file? I am not sure how best to move forward to test with the sensors I have.

thanks for any pointers..
-mark

**Comment by AlCalzone** [comment, marked as answer]
Since it at least has a different label, I would create a new file. We can cut down on the duplication with templates if that is necessary.

**Reply by mnaz** [reply]
ok I will start a pull-request and give it a shot. thanks
